### PR TITLE
Added metadata to glyphs.py, multi_legend.py, multi_scale.py

### PR DESCRIPTION
**Pull request by Yash-Pratapwar**
Added metadata to bokeh\examples\plotting\glyphs.py, bokeh\examples\plotting\multi_legend.py, bokeh\examples\plotting\multi_scale.py

This addresses issue #11765

### CODE REVIEW

**Comment by bryevdv** [comment]
@mattpap I really want all these outside contributions to land on `branch-3.0` and then be forward-ported. Given that you are the only person who will likely be active on `branch-3.1` for the immediate future, I think that the default should be reverted for now.

**Comment by bryevdv** [comment]
Thanks for the PR @Yash-Pratapwar currently there are just some linter errors:
```
E         File contains trailing whitespace: examples/plotting/glyphs.py, line 2.
E         File contains trailing whitespace: examples/plotting/multi_legend.py, line 6.
E         File contains trailing whitespace: examples/plotting/multi_scale.py, line 1.
E         File contains trailing whitespace: examples/plotting/multi_scale.py, line 5.
E         examples/plotting/glyphs.py:6:166: E501 line too long (529 > 165 characters)
```

It's helpful to configure your editor or IDE to automatically trim trailing whitespace on save, if possible. Otherwise you can run the codebase tests locally with
```
pytest tests/codebase
```
For the line too long, you can shorten it somehow if that is feasible, or else add a comment `# noqa` at the end

**Comment by Yash-Pratapwar** [comment]
> Thanks for the PR @Yash-Pratapwar currently there are just some linter errors:
> 
> ```
> E         File contains trailing whitespace: examples/plotting/glyphs.py, line 2.
> E         File contains trailing whitespace: examples/plotting/multi_legend.py, line 6.
> E         File contains trailing whitespace: examples/plotting/multi_scale.py, line 1.
> E         File contains trailing whitespace: examples/plotting/multi_scale.py, line 5.
> E         examples/plotting/glyphs.py:6:166: E501 line too long (529 > 165 characters)
> ```
> 
> It's helpful to configure your editor or IDE to automatically trim trailing whitespace on save, if possible. Otherwise you can run the codebase tests locally with
> 
> ```
> pytest tests/codebase
> ```
> 
> For the line too long, you can shorten it somehow if that is feasible, or else add a comment `# noqa` at the end

Sure, will do the required changes.

File: examples/plotting/glyphs.py [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+Such as annular wedge, annulus, arc, bezier, circle, ellipse, Hbar, line etc.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.annular_wedge, bokeh.plotting.figure.annulus,
+    bokeh.plotting.figure.arc, bokeh.plotting.figure.bezier, bokeh.plotting.figure.circle,
+    bokeh.plotting.figure.ellipse, bokeh.plotting.figure.hbar, bokeh.plotting.figure.line,
+    bokeh.plotting.figure.multi_line, bokeh.plotting.figure.multi_polygon,
+    bokeh.plotting.figure.patch, bokeh.plotting.figure.patches, bokeh.plotting.figure.quad,
+    bokeh.plotting.figure.quadratic, bokeh.plotting.figure.ray, bokeh.plotting.figure.rect,
+    bokeh.plotting.figure.scatter
```
Comment (bryevdv) [inline examples/plotting/glyphs.py:11]: The extension directive is not smart enough to handle multi-lines. We can only have a single line of `:apis:` They can be fairly long line, within reason, even using `# noqa` This is probably too many. Please pare down to the most interesting/important and put on a single line (here and below)

**Comment by Yash-Pratapwar** [comment]
Sure, will do that.

**Comment by bryevdv** [comment]
@Yash-Pratapwar Do yo intend to continue working on this PR?

**Comment by Yash-Pratapwar** [comment]
@bryevdv Yes I intend to work on this PR, so sorry I was caught up with all the college work.

**Comment by Yash-Pratapwar** [comment]
@bryevdv I ran the git pull command to sync the changes on the remote repo, it integrated the changes of branch 3.1, now it's giving an error while I try to push the chnages to my remote branch. Should I start anew, with new fork of the branch 3.1? or create a new branch from branch 3.0 and do the required changes and create a new PR?

**Comment by bryevdv** [comment]
@Yash-Pratapwar The base branch for the PR also needs to be changed in GitHub. I have updated the base branch to `branch-3.1` above (future ref: the "edit" button next to the title controls this)

**Comment by Yash-Pratapwar** [comment]
> @Yash-Pratapwar What command, and what error? If you've done a rebase, you will need to "force" push the changes to GitHub (`git push -f`)

No, I haven't done a rebase. The base branch was causing the error, since you changed the base branch the error is solved, thanks. I have pushed the required changes.

File: examples/plotting/glyphs.py (hunk 1 above)
Comment (Yash-Pratapwar) [reply examples/plotting/glyphs.py]: Did the required changes.

File: examples/plotting/multi_scale.py [hunk 2]
```diff
@@ -1,3 +1,13 @@
+'''This example shows a graph with multiple scales on y-axis.
+This can be used to depict values in two different units.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.plotting.figure.circle,
+    bokeh.plotting.figure.add_layout
```
Comment (bryevdv) [inline examples/plotting/multi_scale.py:6]: This breaks the docs build. All the APIs have to go on a single line

File: examples/plotting/multi_scale.py [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+    :refs: :ref:`userguide_plotting_figure`
+    :keywords: multiple, scale, graph
```
Comment (bryevdv) [inline examples/plotting/multi_scale.py:8]: I don't think "multiple" is a good keyword here, that could refer to nearly anything

**Comment by Yash-Pratapwar** [comment]
Understood, will do the required changes.

File: examples/plotting/multi_scale.py (hunk 3 above)
Comment (Yash-Pratapwar) [reply examples/plotting/multi_scale.py:8]: Ok, will remove those.

File: examples/plotting/multi_scale.py (hunk 2 above)
Comment (Yash-Pratapwar) [reply examples/plotting/multi_scale.py:6]: Yes, the guidelines said to have a maximum of 80 characters in a line so I wrote it in the new line, will do the required changes.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Yash-Pratapwar !

**Pull request merged**